Sample Data/Custom/DNS_Logs_CL.json

```diff
@@ -1,145 +0,0 @@
-[
```

Comment:
why we are removing this?

Reply:
I removed that because it was obsolete.
Throughout the solution we use the table called `NXLog_DNS_Server_CL` and for some time any references to `DNS_Logs_CL` have been removed. That old sample file was introduced back in #1280 
For `NXLog_DNS_Server_CL` we have sample logs already, and since `DNS_Logs_CL` is not used anywhere anymore, IMHO there is no need for any sample logs for it.

Reply:
@jszigetvari-nxlog lets not remove this file. This file seems to have sample data relevant for NXLog too. Let's keep it as is.

Reply:
@vakohl I believe I can safely say that it is not used or referenced anywhere from our side. But if you'd like, I can put it back.

Reply:
Sample logs are not referenced directly, these are useful for future requirements like troubleshooting or verifying parser. If incase there any need comes where we have to explore different available logs, these would be useful. I would keep it as is.

Reply:
All right, I have re-added that file.